modules/migration-creating-migration-plan-cam.adoc

```diff
@@ -33,37 +33,48 @@ The *Plan name* can contain up to 253 lower-case alphanumeric characters (`a-z,
 . Select the projects to be migrated and click *Next*.
 . Select *Copy* or *Move* for the persistent volume (PV):
 
-* *Copy* copies the data in a source cluster's PV to the replication repository and then restores it on a newly created PV, with similar characteristics, in the target cluster.
+. Select a *Source cluster*, a *Target cluster*, and a *Repository*, and click *Next*.
+. In the *Namespaces* screen, select the projects to be migrated and click *Next*.
+. In the *Persistent volumes* screen, select *Copy* or *Move* for the PVs:
+
+* *Copy* copies the data from the PV of a source cluster to the replication repository and then restores it on a newly created PV, with similar characteristics, in the target cluster.
 +
-Optional: You can verify data copied with the file system method by selecting *Verify copy*. This option generates a checksum for each source file and checks it after restoration. The operation significantly reduces performance.
+If you specified a route to an image registry when you added the source cluster to the web console, you can migrate data directly from the source cluster to the target cluster.
```

Comment:
@apinnick I am a little confused about this words. I guess the meaning is 
"If you specified **a route of image registry** when you add the source cluster to the web console, you can migrate the **referenced images** directly from source cluster to the target cluster"